src/nci_util.c

```diff
@@ -413,6 +413,13 @@ nci_parse_iso_dep_poll_a_param(
                     GDEBUG("  T1 =%s", buf->str);
                     g_string_free(buf, TRUE);
                 }
+                GDEBUG("  T0 = 0x%02x", param->t0);
+                if (param->t0 & NFC_T4A_ATS_T0_A)
+                    GDEBUG("  TA = 0x%02x", param->ta);
+                if (param->t0 & NFC_T4A_ATS_T0_B)
+                    GDEBUG("  TB = 0x%02x", param->tb);
+                if (param->t0 & NFC_T4A_ATS_T0_C)
+                    GDEBUG("  TC = 0x%02x", param->tc);
```

Comment:
`{}` please, e.g.
```c
                if (param->t0 & NFC_T4A_ATS_T0_C) {
                    GDEBUG("  TC = 0x%02x", param->tc);
                }
```
I know this is a bit old-fashioned and modern compilers would probably produce a warning if a macro after `if` gets expanded into multiple statements, but still...

Reply:
Thanks for the review, I have added {} for correct macro expansion